components/badges/badge.css

```diff
@@ -0,0 +1,92 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&family=Roboto:ital,wght@0,300;0,400;0,500;0,700;0,900;1,100&display=swap');
+
+
+
+.badge-container{
+position: relative;
+}
+
+.online-img-badge{
+    background-color:rgb(38, 207, 38);
+    border-radius:50%;
+    border:0.3rem solid white;
+    position: absolute;
+
+}
+.offline-img-badge{
+    background-color:red;
+    border-radius:50%;
+    border:0.3rem solid white;
+    position: absolute;
+}
+.badge-container .avatar-big ~ .online-img-badge {
```

Comment:
Please try to maintain a consistent intentation.

Reply:
Hi! can you please elaborate on it, didn't quite get what you want to say🥲

Reply:
I meant the intenration from left in line 25, 26, 27 etc.. Just a suggestion. Press Alt + Shift + F to format the file

Reply:
okayy sure!